src/MangaJet/app/src/main/java/com/mangajet/mangajet/ui/search/SearchFragment.kt

```diff
@@ -54,7 +54,7 @@ class SearchFragment : Fragment() {
             binding.searchViewList.setOnItemClickListener{ parent, view, position, id ->
                 val intent = Intent(frag, AboutMangaActivity::class.java)
                 MangaJetApp.currentManga = searchViewModel.mangas[id.toInt()]
-                startActivity(intent)}
+                startActivityForResult(intent, MangaJetApp.SEARCH_BY_TAG)}
```

Comment:
Wait a minute... Why you put MangaJetApp.SEARCH_BY_TAG in every opening of AboutMangaActivity from Search fragment?

Reply:
Because its a code of result then activity closed. When activity, which started from 'startActivityForResult()', closes, it will call 'onActivityReturn()' override. In this override we need correct execution of every returned code, which we want. So i thought it will be good practise to use some our code system into MangaJetProject.